applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -474,9 +474,15 @@ get_fs_app(Node, UUID, JObj, <<"set">>) ->
             CallVars = kz_json:to_proplist(kz_json:get_json_value(<<"Custom-Call-Vars">>, JObj, kz_json:new())),
             AppVars = kz_json:to_proplist(kz_json:get_json_value(<<"Custom-Application-Vars">>, JObj, kz_json:new())),
 
+            Command = case kz_json:get_boolean_value(<<"Export-All">>, JObj) of
```

Comment:
this can return `undefined` and will throw a function_clause exception.
i would prefer if you move this to function, something like
```
Command  = get_set_command_action(JObj),

...
get_set_command_action(JObj) ->
     case get_set_action(kz_json:get_boolean_value(<<"Export-All">>, JObj, 'false') of
            'true' -> <<"kz_export">>;
             'false' -> <<"kz_multiset">>
     end.

```

Reply:
OK, broke that out into it's own function. Good catch on the default value